mkdocs/docs/HPC/jupyter.md

```diff
@@ -0,0 +1,138 @@
+
+## Using Jupyter Notebooks on the HPC
+
+### Launching a notebook using the web portal
+
+Through the [HPC-UGent web portal](web_portal.md) you can easily start a Jupyter notebook on a workernode, via the *Jupyter Notebook* button under the *Interactive Apps* menu item.
+
+<center>
+![image](img/ood_start_jupyter.png)
+</center>
+
+After starting the Jupyter notebook using the *Launch* button, you will see it being added in state *Queued* in the overview of interactive sessions (see *My Interactive Sessions* menu item):
+
+<center>
+![image](img/ood_jupyter_queued.png)
+</center>
+
+When your job hosting the Jupyter notebook starts running, the status will first change the *Starting*:
+
+<center>
+![image](img/ood_jupyter_starting.png)
+</center>
+
+and eventually the status will change to *Running*, and you will be able to connect to the Jupyter environment using the blue *Connect to Jupyter* button:
+
+<center>
+![image](img/ood_jupyter_running.png)
+</center>
+
+This will launch the Jupyter environment in a new browser tab, where you can open an existing notebook by navigating to the directory where it is located and clicking it. You can also create a new notebook by clicking on `File`>`New`>`Notebook`:
+
+<center>
+![image](img/ood_jupyter_new_notebook.png)
+</center>
+
+### Using extra Python packages
+
+A number of Python packages are readily available in modules on the HPC. To illustrate how to use them in a Jupyter notebook, we will make use of an example where we want to use numpy in our notebook.
+The first thing we need to do is finding the modules that contain our package of choice. For numpy, this would be the SciPy modules.
+
+To find the appropriate modules, it is recommended to use the shell within the web portal under `Clusters`>`>_login Shell Access`.
+
+<center>
+![image](img/ood_jupyter_open_shell.png)
+</center>
+
+We can see all available versions of the SciPy module by using `module available SciPy`:
+
+```shell 
+$ module available Scipy
+
+------------------ /apps/gent/RHEL8/zen2-ib/modules/all ------------------
+    SciPy-bundle/2022.05-foss-2022a    SciPy-bundle/2023.11-gfbf-2023b (D)
+    SciPy-bundle/2023.07-gfbf-2023a
+
+  Where:
+   D:  Default Module
+...
+```
+
+Not all modules will work for every notebook, we need to use the one that uses the same toolchain as the notebook we want to launch. To find that toolchain, we can look at the `JupyterNotebook version` field when creating a notebook. In our example `7.2.0` is the version of the notebook and `GCCcore 13.2.0` is the toolchain used.
+
+<center>
+![image](img/ood_jupyter_version.png)
+</center>
+
+Most modules' names end with the version of the toolchain used by the module (for example `plotly.py/5.18.0-GCCcore-13.2.0`), however for our SciPy module, that is not the case.
```

Comment:
```suggestion
Module names include the toolchain that was used to install the module (for example `GCCcore-13.2.0` in `plotly.py/5.18.0-GCCcore-13.2.0`).
```

This bit needs to be tweaked a bit, since `gfbf-2023a` is also a toolchain, and it's compatible with `GCCcore/13.2.0`, see also https://docs.hpc.ugent.be/troubleshooting/#module-conflicts